pilot/pkg/config/kube/crdclient/client.go

```diff
@@ -204,6 +207,7 @@ func (cl *Client) HasSynced() bool {
 
 // SyncAll syncs all the objects during bootstrap to make the configs updated to caches
 func (cl *Client) SyncAll() {
+	cl.beginSync.Store(true)
```

Comment:
related to previous PR: why do we parallelized cl.kinds? Isn't it 100% CPU bound? does it actually speed things up?

Reply:
There are many cpu cores,  and the go routines scheduling take charge of mapping them to kernel thread and then scheduled by kernel. The cpu can exceed 100%, but it does help speed up.